tests/RequestBuilderTest.php

```diff
@@ -9,37 +9,151 @@
 
 namespace Mailgun\Tests;
 
+use Http\Message\MultipartStream\MultipartStreamBuilder;
+use Http\Message\RequestFactory;
 use Mailgun\RequestBuilder;
+use PHPUnit\Framework\TestCase;
+use PHPUnit_Framework_MockObject_MockObject as MockObject;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\StreamInterface;
 
-class RequestBuilderTest extends \PHPUnit_Framework_TestCase
+/**
+ * TODO request factory instantiate should be tested somehow. For reason something seems to be wrong with realization
```

Comment:
The only way to write test without changing logic in base class is to keep old output tests. But I think It wouldn't be correct. What do you think?